### ISSUE TITLE
PDP Controller not working

### ISSUE THREAD

**Issue by J2Kbr** [J2Kbr/TitanTwo#82]
```
[USB DESCRIPTOR]
12           bLength: 18
01           bDescriptorType: 1 - DEVICE
00 02        bcdUSB: 2.00
FF           bDeviceClass: 255 (FF)
47           bDeviceSubClass: 71 (47)
D0           bDeviceProtocol: 208 (D0)
40           bMaxPacketSize0: 64
6F 0E        idVendor: 0E6F
63 01        idProduct: 0163
02 01        bcdDevice: 1.02
01           iManufacturer: 1
02           iProduct: 2
03           iSerialNumber: 3
01           bNumConfigurations: 1

09           bLength: 9
02           bDescriptorType: 2 - CONFIGURATION
40 00        wTotalLength: 64
02           bNumInterfaces: 2
01           bConfigurationValue: 1
00           iConfiguration: 0
A0           bmAttributes: 160 (Bus Powered, Remote Wakeup)
FA           MaxPower: 500mA

09             bLength: 9
04             bDescriptorType: 4 - INTERFACE
00             bInterfaceNumber: 0
00             bAlternateSetting: 0
02             bNumEndpoints: 2
FF             bInterfaceClass: 255 (FF)
47             bInterfaceSubClass: 71 (47)
D0             bInterfaceProtocol: 208 (D0)
00             iInterface: 0

07               bLength: 7
05               bDescriptorType: 5 - ENDPOINT
81               bEndpointAddress: 1 IN
03               bmAttributes: 3 (Interrupt, None, Data)
40 00            wMaxPacketSize: 64
04               bInterval: 4

07               bLength: 7
05               bDescriptorType: 5 - ENDPOINT
01               bEndpointAddress: 1 OUT
03               bmAttributes: 3 (Interrupt, None, Data)
40 00            wMaxPacketSize: 64
04               bInterval: 4

09             bLength: 9
04             bDescriptorType: 4 - INTERFACE
01             bInterfaceNumber: 1
00             bAlternateSetting: 0
00             bNumEndpoints: 0
FF             bInterfaceClass: 255 (FF)
47             bInterfaceSubClass: 71 (47)
D0             bInterfaceProtocol: 208 (D0)
00             iInterface: 0

09             bLength: 9
04             bDescriptorType: 4 - INTERFACE
01             bInterfaceNumber: 1
01             bAlternateSetting: 1
02             bNumEndpoints: 2
FF             bInterfaceClass: 255 (FF)
47             bInterfaceSubClass: 71 (47)
D0             bInterfaceProtocol: 208 (D0)
00             iInterface: 0

07               bLength: 7
05               bDescriptorType: 5 - ENDPOINT
02               bEndpointAddress: 2 OUT
01               bmAttributes: 1 (Isochronous, None, Data)
E0 00            wMaxPacketSize: 224
01               bInterval: 1

07               bLength: 7
05               bDescriptorType: 5 - ENDPOINT
83               bEndpointAddress: 3 IN
01               bmAttributes: 1 (Isochronous, None, Data)
80 00            wMaxPacketSize: 128
01               bInterval: 1

[STRING 1 DESCRIPTOR]
Performance Designed Products

[STRING 2 DESCRIPTOR]
Legendary Collection: Deliverer of Truth

[STRING 3 DESCRIPTOR]
000041AC4F74732B
```
Note: missing the Bulk Interface (compared with official XB1 controller)

**Comment by J2Kbr** [comment]
Requested more information on the forums:
https://www.consoletuner.com/forum/viewtopic.php?p=54095#p54095

**Comment by J2Kbr** [comment]
> 1. If you have the Titan Two only connect to PC and the PDP controller on Input-A, the Gtuner's Device Monitor register the button presses?
> 
> 2. What is the console you are using this controller? (PS4, XB1)

> 1. YES
> 
> 2. XB1

**Comment by J2Kbr** [comment]
Request more one test:

1. Titan Two connected to Xbox One, as normal;
2. A original Xbox One controller on INPUT-B;
3. The PDP controller on INPUT-A.

Does the PDP controller works?

**Comment by J2Kbr** [comment]
Fixed on firmware 0.98B-43
